jira/client.py

```diff
@@ -991,9 +991,11 @@ def group_members(self, group):
 
         result = {}
         for user in r['users']['items']:
-            result[user['name']] = {'fullname': user['displayName'], 'email': user.get('emailAddress', 'hidden'),
-                                    'active': user['active']}
-        return result
+            result[user['key']] = {'key': user['key'],
+                                   'fullname': user['displayName'],
```

Comment:
Do we want to also include `user['name']` in the dict, since this is no longer returned as the key?